src/ros_filter.cpp

```diff
@@ -1712,7 +1716,7 @@ namespace RobotLocalization
       // Get latest state and publish it
       nav_msgs::Odometry filteredPosition;
 
-      if (getFilteredOdometryMessage(filteredPosition))
+      if (!disabled_ && getFilteredOdometryMessage(filteredPosition))
```

Comment:
Not sure this is the best place/way to disable the filter:

1. `freqDiag.tick()` would be disabled as well, so we won't have diagnostics for the `HeaderlessTopicDiagnostic`. This would make the diagnostic error, I guess. Maybe we should tick when disabled ?!

2. The code `if (publishAcceleration_ && getFilteredAccelMessage(filteredAcceleration))`, that publishes accelerations, maybe should be inside this if, or disabled as well.